daemon/networkdriver/bridge/driver.go

```diff
@@ -222,11 +222,18 @@ func InitDriver(config *Config) error {
 		bridgeIPv6Addr = networkv6.IP
 	}
 
+	if config.EnableIptables {
+		iptables.FirewalldInit()
+	}
+
 	// Configure iptables for link support
 	if config.EnableIptables {
 		if err := setupIPTables(addrv4, config.InterContainerCommunication, config.EnableIpMasq); err != nil {
 			logrus.Errorf("Error configuing iptables: %s", err)
 			return err
+		} else {
```

Comment:
Similar nit as @icecrime; I think the `else` is redundant, because of the `return err` above (same goes for the two other "else" added in this file)